src/implementation/Server/GRPCServer/actor.ts

```diff
@@ -25,18 +26,18 @@ export default class GRPCServerActor implements IServerActor {
   }
 
   deactivateActor(_actorType: string, _actorId: string): Promise<void> {
-    throw new Error("GRPC is currently not supported.");
+    throw new GRPCNotSupportedError();
   }
 
   init(): Promise<void> {
-    throw new Error("GRPC is currently not supported.");
+    throw new GRPCNotSupportedError();
   }
 
   getRegisteredActors(): Promise<string[]> {
-    throw new Error("GRPC is currently not supported.");
+    throw new GRPCNotSupportedError();
   }
 
   registerActor<T extends AbstractActor>(_cls: Class<T>): Promise<void> {
-    throw new Error("GRPC is currently not supported.");
```

Comment:
There might be more such errors now introduced in some PRs like #485

Reply:
Please forgive me, but could you elaborate??

Reply:
I've searched the repo and there doesn't seem to be anymore files with those errors or similar errors

Reply:
One such example is https://github.com/shubham1172/js-sdk/blob/main/src/implementation/Client/GRPCClient/workflow.ts

Reply:
okay, I have found and updated it. 
Quick question0- Do i create a new folder `errors` in the `test` directory to place the unit test files or do i just add them to an existing folder in the same `test` directory

Reply:
They should be a part of https://github.com/dapr/js-sdk/tree/main/test/unit/<an-existing-folder> to test if components return the correct error, and also in https://github.com/dapr/js-sdk/tree/main/test/unit/errors to test the error itself.